lib/libzfs/libzfs_util.c

```diff
@@ -596,27 +596,47 @@ zfs_strdup(libzfs_handle_t *hdl, const char *str)
  * Convert a number to an appropriately human-readable output.
  */
 void
-zfs_nicenum(uint64_t num, char *buf, size_t buflen)
+__zfs_nicenum(uint64_t num, char *buf, size_t buflen,
```

Comment:
sorry for the confusion, zfs_nicenum is base 2, so zfs_nicenum10 is my idea for base 10